src/nlp/arrow_dataset.py

```diff
@@ -379,9 +380,11 @@ def _getitem(
             if format_type is not None:
                 if format_columns is None or key in format_columns:
                     if format_type == "pandas":
-                        outputs = self._data[key].to_pandas()
+                        outputs = self._data[key].to_pandas(zero_copy_only=False)
```

Comment:
Is there an issue with pandas too ?

Reply:
Did that by precaution but maybe not indeed